app/models/post.rb

```diff
@@ -2,4 +2,8 @@ class Post < ApplicationRecord
   belongs_to :user, counter_cache: true
   has_many :comments
   has_many :likes
+
+  def last_five_comments
+    comments.limit(5).order(created_at: :desc)
+  end
```

Comment:
- [ ] Please work on updating the posts counter method you can do something like:
```ruby
def update_post_counter
  author.increment!(:posts_counter)
end
```

Reply:
### Optional
- [ ] I suggest that you use [Rails active callbacks](https://guides.rubyonrails.org/active_record_callbacks.html), `after_save` to call the `#update_post_counter` method every time a new post has been saved to the database, then make the `update_post_counter` method private

Reply:
- [ ] In the `comment` model you are supposed to add a method that updates the comments counter for a post you can do:
```ruby
def update_comments_counter
  post.increment!(:comments_counter)
end
```

Reply:
## Optional
- [ ] I suggest that you use [Rails active callbacks](https://guides.rubyonrails.org/active_record_callbacks.html), `after_save` to call the `#update_comments_counter` method every time a new `comment` has been saved to the database, then make the `update_comments_counter` method private

Reply:
- [ ] In the `like` method you are supposed to add a method that updates the likes counter for a post, you can do:
```ruby
def update_likes_counter
  post.increment!(:likes_counter)
end
```

Reply:
## Optional
- [ ] I suggest that you use [Rails active callbacks](https://guides.rubyonrails.org/active_record_callbacks.html), `after_save` to call the `#update_likes_counter` method every time a new `like` has been saved to the database, then make the `update_likes_counter` method private

Reply:
Thank you so much @DeliceLydia, I'll implement the changes you've so modestly requested.